config/initializers/doorkeeper.rb

```diff
@@ -100,3 +100,4 @@
   # set to true if you want this to be allowed
   # wildcard_redirect_uri false
 end
+Doorkeeper.configuration.token_grant_types << "password"
```

Comment:
As per Doorkeeper's documentation: https://github.com/doorkeeper-gem/doorkeeper/wiki/Using-Resource-Owner-Password-Credentials-flow

<code>For Doorkeeper 2.1+, you'll need to add password as an acceptable grant type. In your initializer after the Doorkeeper.configure block, add:

Doorkeeper.configuration.token_grant_types << "password"</code>

just enabling grant_flows wouldn't work.